applications_tests/gls_sharp_navier_stokes_3d/rbf_sphere_mms.output

```diff
@@ -6,17 +6,17 @@ Running on 1 MPI rank(s)...
 *****************************************************************
 Steady iteration :        1/3
 *****************************************************************
-div u : 1.241e-08
-error at the weak BC : 0.3751
-L2 error velocity : 0.3046 L2 error pressure: 1.342
+div u : 0
+error at the weak BC : 0
+L2 error velocity : 0 L2 error pressure: 0
```

Comment:
Why is this all zero now? it seems like there is no solution at this iteration?

Reply:
It's because all the cells are either overconstrained or cut. I could remove this refinement level or leave it, as it shows that the solver doesn't crash in that case.

Reply:
Nah it's good for me, as long as you understand why it's doing that it's ok